source/03-components/button/button.yml

```diff
@@ -4,6 +4,7 @@ text_demo: 'Link Button'
 is_demo: true
 is_button_tag: true
 is_disabled: false
+data_attributes: ''
```

Comment:
Is this data_attributes bit left over from a previous approach?

Reply:
Yep, just removed it here too.